packages/frontend/core/src/mixpanel/events.ts

```diff
@@ -3,7 +3,36 @@
 
 // SECTION: app events
 type GeneralEvents = 'openMigrationDataHelp' | 'export';
-type CmdkEvents = 'quickSearch';
+type CmdkEvents =
+  | 'quickSearch'
+  | 'recentDocs'
+  | 'searchResultsDocs'
+  | 'copyPrivateLink'
+  | 'goBack'
+  | 'goForward'
+  | 'findInPage'
+  | 'openSettings'
+  | 'changeLanguage'
+  | 'changeColorMode'
+  | 'changeFontStyle'
+  | 'changeClientBorderStyle'
+  | 'changeFullWidthLayout'
+  | 'changeNoisyBackgroundOnTheSidebar'
+  | 'changeTranslucentUIOnTheSidebar'
+  | 'goTo'
+  | 'viewInfo'
+  | 'toggleFavorite'
+  | 'switchPageMode'
+  | 'export'
+  | 'moveToTrash'
+  | 'restoreFromTrash'
+  | 'revealDocHistoryModal'
+  | 'save'
+  | 'restartToUpgrade'
+  | 'toggleSidebar'
+  | 'whatNew'
+  | 'contactUs'
+  | 'openChangelog';
```

Comment:
control the amount of event and reuse them, for example all setting changes and reuse `changeAppSetting()`

Reply:
I sorted out the event types in this PR and deleted some duplicate events.